usr/share/lib/img_proof/tests/SLES/conftest.py

```diff
@@ -140,6 +140,10 @@
 SLE_12_SP3_MODULES = SLE_12_SP2_MODULES
 
 SLE_12_SP3_SAP = [
+    'SLE-Live-Patching12-SP4-Debuginfo-Pool',
```

Comment:
SP3?

Reply:
That's what happens when I find uncommitted work lying around weeks later :( . Fixed now.